test/membrane/data-integrity.spec.js

```diff
@@ -0,0 +1,34 @@
+import createVirtualEnvironment from '@locker/near-membrane-dom';
+
+describe('Data', () => {
+    it('encodes blobs from typed arrays correctly', (done) => {
+        const env = createVirtualEnvironment(window, window, { endowments: { done, expect } });
+        env.evaluate(`
+            const a = new Uint8Array([97, 98, 99]);
+            // Demonstrates that the value is still a number, as expected.
+            expect(typeof a[0]).toBe('number');
+            expect(typeof a[1]).toBe('number');
+            expect(typeof a[2]).toBe('number');
+
+            // As far as I can tell, the type doesn't matter
+            const b = new Blob([a], { type: 'application/octet-stream' });
```

Comment:
This should work fine now because `a` is a blue object representing a typed array under the hood, and `Blob` is a blue API as well, so they should work fine together.